src/model/GameDto.js

```diff
@@ -0,0 +1,21 @@
+const Sequelize = require('sequelize');
+
+let gameModal = function (sequelize) {
+    var GameDto = sequelize.define('Game', {
+            Id:{
+              type: Sequelize.INTEGER,
+              autoIncrement: true,
+              primaryKey: true,
+              unique: true
+            },
+            PlayerId:{
+              type: Sequelize.STRING,      
```

Comment:
a game should have 2 players, so in your model you should save both ids in order to identify them correctly